app/lib/user_with_organisation_context.rb

```diff
@@ -13,6 +13,12 @@ class << self
     def primary_key
       "id"
     end
+
+    delegate :composite_primary_key?, :has_query_constraints?, to: :model
+
+    def model
+      User
+    end
```

Comment:
I think the following two methods could use this as a helper (to remove other hard-coded refs to `User`).